saleor/graphql/product/types.py

```diff
@@ -42,6 +42,16 @@ class StockAvailability(graphene.Enum):
     OUT_OF_STOCK = 'OUT_OF_STOCK'
 
 
+class ProductOrderField(graphene.Enum):
```

Comment:
Graphene allows adding descriptions for enum values as well - https://docs.graphene-python.org/en/latest/types/enums/#value-descriptions

Here I would add:
- name - "Sort products by name"
- price = "Sort products by price"